web_client/js/imageViewerWidget/base.js

```diff
@@ -8,6 +8,8 @@ girder.views.ImageViewerWidget = girder.View.extend({
         }).done(_.bind(function (resp) {
             this.levels = resp.levels;
             this.tileSize = resp.tileSize;
```

Comment:
I think we should just get rid of ``tileSize`` everywhere now. A viewer can always choose to fail to initialize if it doesn't support non-square tiles.